@ecomplus/storefront-components/src/html/TheCart.html

```diff
@@ -0,0 +1,128 @@
+<div class="cart">
+    <transition-group enter-active-class="animated fadeInDown">
```

Comment:
```suggestion
  <transition-group enter-active-class="animated fadeInDown">
```

A identação ficou errada daqui pra baixo